components/shapediver/parameter/ParameterWrapperComponent.module.css

```diff
@@ -0,0 +1,3 @@
+.wrapperModified {
+	background-color: var(--mantine-color-blue-light);
```

Comment:
@LemToUp let's use a variable that references a shade of the primary color, otherwise it will look strange if users use a different one

Reply:
@LemToUp I tested this locally, and the color only works for me if I add `!important`.